doors.lua

```diff
@@ -14,323 +14,379 @@
 -- will the authors be held liable for any damages arising from the use of this content.
 
 
-local doors = {
-	-- Black door
-	{"scifi_nodes:door2a","scifi_nodes:door2b","scifi_nodes:door2c","scifi_nodes:door2d","2","black", "doors:door_steel", "scifi_nodes_door_mechanic"},
-	-- White door
-	{"scifi_nodes:door3a","scifi_nodes:door3b","scifi_nodes:door3c","scifi_nodes:door3d","3","white", "doors:door_glass", "scifi_nodes_door_normal"},
-	-- Green door
-	{"scifi_nodes:door4a","scifi_nodes:door4b","scifi_nodes:door4c","scifi_nodes:door4d","4","green", "doors:door_wood", "scifi_nodes_door_mechanic"},
-	-- DOOM door
-	{"scifi_nodes:door1a","scifi_nodes:door1b","scifi_nodes:door1c","scifi_nodes:door1d","1","Doom", "doors:door_obsidian_glass", "scifi_nodes_door_mechanic"}
-}
+-- Retrieving mod settings
+local scifi_nodes = {}
+scifi_nodes.doors_open_with_mesecon_only = minetest.settings:get_bool("scifi_nodes.doors_open_with_mesecon_only", true)
+print("Ici !"..dump(scifi_nodes))
```

Comment:
Stray print() from a debug-session? :smile:

Reply:
Yep : a stray print xD !

And there's a setting ;-)